### PR TITLE
Fix warnings in project.json

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb \ @davidfowl 

Using this as a template. i'll bulk update repos if this looks good.

**Comment by davidfowl** [comment]
Change:

```JSON
{
   "content": [ ... ]
}
```

to

```JSON
{
   "publishOptions": {
        "include": [ ... ]
   }
}
```

File: samples/EmbeddedViewSample.Web/project.json [hunk 1]
```diff
@@ -1,9 +1,11 @@
 {
-  "compilationOptions": {
+  "buildOptions": {
     "emitEntryPoint": true,
-    "preserveCompilationContext": true
+    "preserveCompilationContext": true,
+    "embed": {
```
Comment (pranavkm) [inline samples/EmbeddedViewSample.Web/project.json]: :eyes:

**Comment by pranavkm** [comment]
Updated

File: samples/ActionConstraintSample.Web/project.json [hunk 2]
```diff
@@ -24,9 +24,11 @@
       }
     }
   },
-  "content": [
-    "web.config"
-  ],
+  "publish": {
```
Comment (ajaybhargavb) [inline samples/ActionConstraintSample.Web/project.json]: publishOptions

**Pull request closed without merging**